pages/common/adb-reboot.md

```diff
@@ -0,0 +1,20 @@
+# adb reboot
+
+> Android Debug Bridge Install: reboot an Android emulator instance or connected Android devices.
+> More information: <https://developer.android.com/tools/adb>.
+
+- Reboot the device normally:
+
+`adb reboot`
+
+- Reboot the device into bootloader mode:
+
+`adb reboot bootloader`
+
+- Reboot the device into recovery mode:
+
+`adb reboot recovery`
+
+- Reboot the device into fastboot mode:
+
+`adb reboot fastboot`
```

Comment:
Now that I look at the manpage, does the fastboot subcommand even exist or do anything?

Reply:
I tested it. Despite not being listed in the man page, it does indeed exist.